src/badge/theme.scss

```
@@ -10,16 +10,20 @@
     float: left; 
 }
 
+.clearfix {
+  clear: both;
+}
+
 :local(.badge-container-small) {
-    padding: 1.5%;
+    padding: 10px;
```

I think we should avoid using `px`?
```suggestion
    padding: 1.5%;
```